.github/workflows/scheduled-docker-builds.yml

```diff
@@ -0,0 +1,39 @@
+name: Scheduled Docker Builds
+
+on:
+  schedule:
+    - cron: "0 2 * * *"
+  workflow_dispatch:
+
+jobs:
+  docker-scheduled-build:
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        project:
+          [
+            mithril-aggregator,
+            mithril-client-cli,
+            mithril-signer,
+            mithril-relay,
+          ]
+
+        include:
+          - project: mithril-client-cli
+            package: mithril-client
```

Comment:
What is the utility of those lines ? There's no `matrix.package` usage below.

Reply:
Indeed, I'll remove this block. It's a leftover from the CI step I used as a reference to create the workflow.